snap/snapcraft.yaml

```diff
@@ -28,19 +27,12 @@ confinement: strict
 parts:
   jhack:
     plugin: python
-    source: https://github.com/PietroPasotti/jhack.git
-    source-branch: main
-    python-packages:
-      # TODO: keep pyproject in sync with these!
-      - typer==0.7.0
-      - ops==1.5.3
-      - rich==13.3.0
-      - parse==1.19.0
-      - asttokens
-      - astunparse
-      - requests==2.29.0
-      - requests-unixsocket==0.3.0
-      - toml
+    source: .
+    override-build: |
+      snapcraftctl build
+      snapcraftctl set-version \
+        $(PYTHONPATH=$SNAPCRAFT_PART_INSTALL/lib/python3.8/site-packages/ \
```

Comment:
I'd rather using `cat | grep` instead of going through python.
```
      VERSION=$(cat ./pyproject.toml | grep -Po 'version = "\K[^"]*')
      snapcraftctl set-version $VERSION
```